src/altinn-app-frontend/src/features/form/data/submit/submitFormDataSagas.ts

```diff
@@ -175,7 +170,7 @@ export function* putFormData(state: IRuntimeState, model: any) {
     if (isIE) {
```

Comment:
We no longer support IE btw, so this can be removed. I dont think the app will even work in IE anymore.

Reply:
Nice! I never noticed this. Removing that branch.